assets/js/gutenberg-plugin.js

```diff
@@ -0,0 +1,167 @@
+import { wp, dtGutenberg } from 'window';
+import PluginIcon from '../img/icon.svg'; // eslint-disable-line no-unused-vars
+
+const { Icon } = wp.components; // eslint-disable-line no-unused-vars
```

Comment:
Look like we haven't added Eslint rules for react yet. I think it's better to do it in this PR  so we don't have to add these comments to display Eslint.

Reply:
@dinhtungdu Any recommendations on the best way to add these rules without impacting other things? We currently use an older version of the `@10up/eslint-config` and I updated that to the latest version (following instructions [here](https://github.com/10up/10up-toolkit/blob/develop/packages/eslint-config/README.md)), but I find that brings over a lot of pretty strict rules, some of which I don't agree with (which is fine, that's just my opinion).

But what this does mean is all the JS files will need a bunch of minor formatting changes applied in order to pass these new rules. Not sure we want all of that cleanup work on this PR, so wondering if there's either a simpler solution for now to add better React eslint rules or if we tackle this in a separate PR, one built around updating our entire build/lint process?

Reply:
I'm fine with (1) resolving those eslint failures in a separate PR and (2) not worrying about those until the next release (aka let's not block this issue/release by those eslint issues).

Reply:
> But what this does mean is all the JS files will need a bunch of minor formatting changes applied in order to pass these new rules.

I didn't think about this. Let's fix the eslint in other PR.